examples/rcbilling-demo/src/pages/paywall/index.tsx

```diff
@@ -91,13 +91,19 @@ export const PackageCard: React.FC<IPackageCardProps> = ({
 const PaywallPage: React.FC = () => {
   const navigate = useNavigate();
   const { purchases, offerings } = usePurchasesLoaderData();
```

Comment:
In order to use the new sdk `getOffering` method we need to update the `usePurchasesLoaderData` hook to accept the `offering_id` parameter.